modules/file-integrity-operator-installing-cli.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-installation.adoc
+
+[id="installing-file-integrity-operator-using-cli_{context}"]
+= Installing the File Integrity Operator using the CLI
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Create a namespace object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-file-integrity
+----
+
+. Apply the file by running:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create the `OperatorGroup` object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: file-integrity-operator
+  namespace: openshift-file-integrity
+spec:
+  targetNamespaces:
+  - openshift-file-integrity
+----
+
+. Create the Subscription object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: file-integrity-operator-sub
+  namespace: openshift-file-integrity
+spec:
+  installPlanApproval: Automatic
+  name: file-integrity
```

Comment:
for the subscription, there is an important field called "channel":
a. Run the following command to set the OpenShift Container Platform major and minor version as an environment variable, which is used as the channel value in the next step.
OC_VERSION=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
b. Create the Subscription object YAML file by running:
----
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: file-integrity-operator-sub
  namespace: openshift-file-integrity
spec:
  channel: "${OC_VERSION}"
  installPlanApproval: Automatic
  name: file-integrity
  source: redhat-operators
  sourceNamespace: openshift-marketplace
----

Reply:
@ahardin-rh Hi, the `name` should be `file-integrity-operator` in above subscription example file otherwise the operator will not be installed.

```
$ oc get packagemanifest |grep file
file-integrity-operator                              Red Hat Operators     118m
```